test/e2e_node/security_context_test.go

```diff
@@ -313,6 +313,66 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		It("should run the container with uid 0", func() {
 			createAndWaitUserPod(0)
 		})
+	})
+
+	Context("When creating a pod with readOnlyRootFilesystem", func() {
+		makeUserPod := func(podName, image string, command []string, readOnlyRootFilesystem bool) *v1.Pod {
+			return &v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: podName,
+				},
+				Spec: v1.PodSpec{
+					RestartPolicy: v1.RestartPolicyNever,
+					Containers: []v1.Container{
+						{
+							Image:   image,
+							Name:    podName,
+							Command: command,
+							SecurityContext: &v1.SecurityContext{
+								ReadOnlyRootFilesystem: &readOnlyRootFilesystem,
+							},
+						},
+					},
+				},
+			}
+		}
+		createAndWaitUserPod := func(readOnlyRootFilesystem bool) string {
+			podName := fmt.Sprintf("busybox-readonly-%v-%s", readOnlyRootFilesystem, uuid.NewUUID())
+			podClient.Create(makeUserPod(podName,
+				"gcr.io/google_containers/busybox:1.24",
+				[]string{"sh", "-c", "touch checkfile && [ -f checkfile ] && echo Found || true"},
```

Comment:
Alternatively, you can simply run `"sh", "-c", "touch checkfile` and see whether the pod succeeded or failed. The upside is that we wouldn't need "logs" to check the state of the pod.